python/snewpy/models/extended.py

```diff
@@ -0,0 +1,107 @@
+from warnings import warn
+
+import numpy as np
+from astropy import units as u
+
+from snewpy.neutrino import Flavor
+from snewpy.models.base import SupernovaModel
+
+
+class ExtendedModel(SupernovaModel):
+    """Class defining a supernova model with a cooling tail extension."""
+
+    def __init__(self, base_model):
+        """Initialize extended supernova model class."""
+        if not isinstance(base_model, SupernovaModel):
+            raise TypeError("ExtendedModel.__init__ requires a SupernovaModel object")
+
+        self.__dict__ = base_model.__dict__.copy()
+        for method_name in dir(base_model):
+            if callable(getattr(base_model, method_name)) and method_name[0] != '_':
+                if method_name == 'get_initial_spectra':
+                    self._get_initial_spectra = getattr(base_model, method_name)
+                else:
+                    setattr(self, method_name, getattr(base_model, method_name))
+        self.t_final = self.time[-1]
+        self.L_final = {flv: self.luminosity[flv][-1] for flv in Flavor}}
```

Comment:
```suggestion
        self.L_final = {flv: self.luminosity[flv][-1] for flv in Flavor}
```
Fix an unfortunate typo